module_3/src/service/User.js

```diff
@@ -0,0 +1,63 @@
+import API from './Api.js';
+import { userModel } from '../model/user.js';
+import { v4 as uuidv4 } from 'uuid';
+import statusCodes from '../../statusCodes.js';
+
+class User extends API {
```

Comment:
User must split into two services - `UserService` or `UserRepository` - which will manage user data
second one must be a `UserApiService` which will handle API requests.
S - Single Responsibility 😄

Reply:
hi, thx for the comments, i updated the code regarding the previous ones, but can't catch what did u mean in this one?  can u rephrase it, or give an example?
For this moment API created as a place to save additional helpers functions. I renamed it and remove extending it by UserService.